_posts/2024-05-14-homebrew-4.3.0.md

```diff
@@ -0,0 +1,104 @@
+---
+title: 4.3.0
+author: MikeMcQuaid
+redirect_from: /blog/4.3.0/
+---
+
+Today, I'd like to announce Homebrew 4.3.0.
+The most significant changes since 4.2.0 are SBOM support, initial attestation verification, new command analytics and uninstall autoremove by default.
+
+Major changes and deprecations since 4.2.0:
+
+- [`brew bottle` will include a basic SPDX file inside the bottle](https://github.com/Homebrew/brew/pull/16594)
+  [and a more comprehensive one after installation](https://github.com/Homebrew/brew/pull/17254).
+  This is to provide support for the widely used SBOM format from Homebrew.
+
+- [If `HOMEBREW_VERIFY_ATTESTATIONS` is set, `brew install` will verify the attestation chain of trust (wording?) when pouring bottles using GitHub's `gh` CLI](https://github.com/Homebrew/brew/pull/17049).
```

Comment:
Not sure about wording either; leaving a comment here to make sure this gets attention and is dealt with before merging.